amundsen-kube-helm/templates/helm/templates/deployment-frontend.yaml

```diff
@@ -0,0 +1,100 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "amundsen.fullname" . }}-{{ .Values.frontEnd.serviceName }}
+  labels:
+    app: {{ template "amundsen.name" . }}
+    component: {{ .Values.frontEnd.serviceName }}
+    chart: {{ template "amundsen.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ template "amundsen.name" . }}
+      component: {{ .Values.frontEnd.serviceName }}
+      release: {{ .Release.Name }}
+  replicas: {{ default 1 .Values.frontEnd.replicas }}
+  template:
+    metadata:
+      {{- with default .Values.podAnnotations .Values.frontEnd.podAnnotations }}
+      annotations:
+{{ toYaml . | indent 8 }}
+      {{- end }}
+      labels:
+        app: {{ template "amundsen.name" . }}
+        component: {{ .Values.frontEnd.serviceName }}
+        release: {{ .Release.Name }}
+    spec:
+      {{- with default .Values.nodeSelector .Values.frontEnd.nodeSelector }}
+      nodeSelector:
+{{ toYaml . | indent 8 }}
+      {{- end }}
+      {{- with default .Values.affinity .Values.frontEnd.affinity }}
+      affinity:
+{{ toYaml . | indent 8 }}
+      {{- end }}
+      {{- with default .Values.tolerations .Values.frontEnd.tolerations }}
+      tolerations:
+{{ toYaml . | indent 8 }}
+      {{- end }}
+      volumes:
+      {{- if .Values.frontEnd.oidcEnabled }}
+        - name: oidc-config
+          secret:
+            secretName: oidc-config
+      {{- end }}
+      containers:
+      - name: {{ .Chart.Name }}-{{ .Values.frontEnd.serviceName }}
+        image: {{ .Values.frontEnd.image }}:{{ .Values.frontEnd.imageTag }}
+        imagePullPolicy: Always
+        ports:
+          - containerPort: 5000
+        env:
+          - name: FRONTEND_BASE
+            value: {{ .Values.frontEnd.fqdn }}
```

Comment:
Where does `.Values.frontEnd.fqdn` come from? Should it also be documented in README.md?

Admittedly pretty new to kube and helm, so this could just be my lack of understanding.

Reply:
Ah, good catch! I missed that one. Updated and added to README.md and values.yaml.